Sources/PublishCLICore/WebsiteRunner.swift

```diff
@@ -28,9 +28,19 @@ internal struct WebsiteRunner {
         """)
 
         serverQueue.async {
+
+            // If the system's Python version is Python 3, use the new `http.server` command.
+            // Otherwise, default to the `SimpleHTTPServer` command that Python 2 uses.
+            var pythonHTTPServerCommand: String
```

Comment:
Rather than including a local variable, I'd make the `systemPythonMajorVersionNumber` property a method instead called `resolvePythonServerCommand()` which simply returns the string directly. There's no need for any code outside of that method to be aware of the Python version, as far as I can tell.